### PR TITLE
Collect not-in-use e2e projects and add them to boskos

### PR DESCRIPTION
**Pull request by krzyzacy**
So that we can have an initial project pool

/assign @fejta

### CODE REVIEW

File: boskos/resources.json [hunk 1]
```diff
@@ -1,5 +1,822 @@
 [
-{"name": "FAKE_PROJECT_1", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_2", "type": "project", "state": "free"},
-{"name": "FAKE_PROJECT_3", "type": "project", "state": "free"}
+  {
```
Comment (fejta) [inline boskos/resources.json]: a) Why are we checking in a state of free? It seems like that will depend on the state of boskos. In other words I think the config format (what things do we want in boskos) to be slightly different from the state format (where is each thing right now?)


b) Would it make more sense to make this config `[ "TYPE": [ "PROJ1", "PROJ2" ]]`, aka:
```
{
  "project": 
    "initial": "dirty",  // so we clean it up before using it
    "names: [
    "ci-kubernetes-e2e-gke-gpu",
    "k8s-jkns-f8n-gce-upg",
    ...
  ],
  "future_type":
  "initial": "who-knows",
  "names": [
    "future-name", "future-other-name", ...
  ]
}
```

Comment (krzyzacy) [inline boskos/resources.json]: Clean before toss into the free pool sounds sane, I can take that.

Reformat config makes me need to write code to parse the input, rather than I can do a raw unmarshal from json to []resources. What if we'll have more per-resource data, for example, I'll soon need to add maybe some tags to classify gce vs gke projects.

Comment (fejta) [inline boskos/resources.json]: Once we have boskos managing 200 jobs, how do you intend to add the 201st one without causing an outage? If the config and state formats are the same this seems dangerous, we're likely to reset everything as free and then cause a bunch of problems.



Change the type from project to gke-project, gce-project, large-project, etc.

Comment (krzyzacy) [inline boskos/resources.json]: when adding new projects it does not do a blind reload, it will compare existing and new projects, add new ones and delete old ones respectively.

**Comment by fejta** [comment]
/lgtm

Please change the initial free state to dirty.

File: boskos/resources.json (hunk 1 above)
Comment (fejta) [inline boskos/resources.json]: Will LGTM, but I hope you understand why this worries me?

Comment (krzyzacy) [inline boskos/resources.json]: ack, will send another PR
